lineman/app/services/modal_service.coffee

```diff
@@ -0,0 +1,25 @@
+angular.module('loomioApp').factory 'ModalService', ($modal, $rootScope) ->
+  new class ModalService
+    openModal: (name, options = {}) ->
+      return console.log "Invalid modal name: #{name}" unless @modals[name]
+      $rootScope.$broadcast 'modalOpened'
+      $modal.open
+        templateUrl: @modals[name].template
+        controller:  @modals[name].controller
+        resolve:     @setLocals name, options
+
+    setLocals: (name, options) ->
+      locals = {}
```

Comment:
I think I need to revisit this when we start passing objects to the modal.

It doesn't affect the start menu, so I'm not bothering with it too hard right now.

The intended effect is to pass resolve objects to the modal based on an options hash.
The code I want to write:
```
ModalService.open('discussionForm', { discussion: $scope.discussion })
```